modules/oadp-converting-to-new-dpa-1-2-0.adoc

```diff
@@ -49,3 +49,8 @@ spec:
 ----
 
 . Wait for the DPA to reconcile successfully.
+
+[NOTE]
+====
+The `fs-backup/restic` timeout is one-hour by default. In OADP 1.3.1 the default is four hours for both Restic and Kopia file system backup.
```

Comment:
1. If the timeout value is being changed for 1.3.1+ versions then  for clarity "In OADP 1.3.1 **and later**" can be used IMO.
2. Instead of using the `fs-backup/restic` parameter, can we just use "Restic"? Because in the next sentence we are using file system backup names (Rustic and Kopia) instead of parameters.
3. The default timeout value for Kopia earlier OADP 1.3.1 was one hour, like Restic? If so, then it can be mentioned in the first sentence.

Reply:
@xenolinux 
Thank you for your good suggestions.